### DISCUSSION TITLE
Features only available in the paid version?

### DISCUSSION THREAD

**Discussion by Shohreh** [osmandapp/OsmAnd#16270] [Q&A] [answered]
Hello,

[On Google](https://play.google.com/store/search?q=osmand&c=apps), I see OsmAnd is available in two version: Free and paid.

Unless I misssed it, [the site](https://osmand.net/) doesn't say what features the paid version adds.

Is there a recent list somewhere so I can tell if I need the paid version?

Thank you.

**Comment by stephan75** [comment, marked as answer]
Maybe have a closer lookk at https://osmand.net/docs/user/purchases/android

**Reply by Shohreh** [reply]
Thank you.
